src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let status = Object.assign({}, state);
+  const result = [];
+
+  actions.forEach(action => {
+    let newState = {};
+
+    if (action.type === 'addProperties') {
```

Comment:
It would be better to use switch for this. See the checklist ;)

Reply:
ready to check :)